.circleci/config.yml

```diff
@@ -7,7 +7,8 @@ version: 2
 jobs:
   build:
     working_directory: ~/test
-    machine: true
+    machine:
+      image: ubuntu-2004:current
```

Comment:
I am just wondering why this change was needed

Reply:
These changes come from re-rendering with the latest conda-smithy. From my understanding, it is just good practice that when you open a PR in a feedstock, you should re-render to make sure all of the common feedstock files are up to date with any new changes conda-forge has had since the last PR/render. Overall, the only files I directly changed (across all of the feedstocks, not just this one) are the meta.yaml, README.md, and CODEOWNERS, everything else is from the re-render called in the comment above